packages/slate-react/src/plugins/after.js

```diff
@@ -595,10 +594,7 @@ function AfterPlugin() {
    */
 
   function renderEditor(props, editor) {
-    const handlers = EVENT_HANDLERS.reduce((obj, handler) => {
-      obj[handler] = editor[handler]
-      return obj
-    }, {})
+    const handlers = editor.getHandlers()
```

Comment:
I like how handlers have been simplified in other parts of this pull request. But I think here it feels strange. This is probably because it's a bit weird that we're rendering `<Content>` here, instead of passing it through as a child in the `Editor.render`. Could you change that? Then there will be no reaching for those handlers.